autogen/variables.tf.tmpl

```diff
@@ -92,6 +92,27 @@ variable "stateful_disks" {
   default = []
 }
 
+#################
+# Stateful IPs
+#################
+variable "stateful_internal_ips" {
```

Comment:
How about adding a flag for `is_external` on the object so we don't have two different variables?

Reply:
Great feedback and incorporate it into next comment